packages/iov-bns/src/util.ts

```diff
@@ -57,10 +57,21 @@ function keyToIdentifier(key: PubkeyBundle): Uint8Array {
   return Uint8Array.from([...algoToPrefix(key.algo), ...key.data]);
 }
 
+/**
+ * Creates an IOV address from a given Ed25519 pubkey and a prefix that represents the network
```

Comment:
Suggest "represents the network kind (ie mainnet or testnet)"

Reply:
Added, thanks